$MirOS: ports/lang/llvm-gcc/patches/patch-gcc_gthr-posix_c,v 1.1 2008/11/11 05:30:59 tg Exp $

	Let this compile with all these Apple locals

--- gcc/gthr-posix.c.orig	Thu Apr  9 23:22:43 2009
+++ gcc/gthr-posix.c	Sun Dec  6 14:16:31 2009
@@ -24,6 +24,9 @@ see the files COPYING3 and COPYING.RUNTI
 <http://www.gnu.org/licenses/>.  */
 
 #include "tconfig.h"
+#include "config.h"
+#include "system.h"
+#include "coretypes.h"
 #include "tm.h"
 # define __gthrw_pragma(pragma) _Pragma (#pragma)
 /* Define so we provide weak definitions of functions used by libobjc only.  */
